server/images.go

```diff
@@ -971,7 +979,8 @@ func PullModel(ctx context.Context, name string, regOpts *registryOptions, fn fu
 		fn(api.ProgressResponse{Status: "removing any unused layers"})
 		err = deleteUnusedLayers(nil, deleteMap)
 		if err != nil {
-			return err
+			slog.Info(fmt.Sprintf("couldn't remove unused layers: %v", err))
```

Comment:
we may want to upgrade this to `slog.Error` given it could start increasing disk usage

Reply:
Sure